e2e/internal-matcher-exposure/__tests__/index.test.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const {expect} = require('expect');
+
+expect.extend({
+  toPass(received, expected) {
+    const {pass} = this.matchers.toEqual(received, expected);
```

Comment:
to make this test pass I have to do `this.matchers.toEqual.call(this, received, expected);` here - i.e. explicitly pass in `this`. any idea what changed to make this test now fail?

Reply:
no clue but I'll take a look

Reply:
Solved this by binding the context to the matchers when I expose them, but doing this way the unit testing started failing.

Later I will try to fix them